src/test/java/org/editorconfig/plugin/maven/model/EndOfLineTest.java

```diff
@@ -13,13 +13,17 @@ class EndOfLineTest {
 
     @ParameterizedTest
     @EnumSource(value = EndOfLine.class, names = "CARRIAGE_RERUN_LINE_FEED", mode = Mode.EXCLUDE)
-    void isSingleCharacter_true(EndOfLine source) {
+    void isSingleCharacter_СrLfEof_True(EndOfLine source) {
```

Comment:
It is everything except the `CRLF`, which is not a single character, so the name of the test is misleading :)